src/modules/Sketch/info.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "sketch",
```

Comment:
I think you are using canvas so please add a `requires` field in the info.json. Please refer to the info.json of text overlay and other modules which use `canvas`.